React/my-app/src/utils/arrayUtils.js

```diff
@@ -0,0 +1,7 @@
+export const bookAvailable = books => {
+  return books.every(book => book.returned_at);
+};
+
+export const userHasTheBook = books => {
+  return books.some(book => book.user.id === window.localStorage.userId);
+};
```

Comment:
I'd move these functions to the component (or another file in it's folder). Utils should be generic.